orcestra/weathermaps.py

```diff
@@ -82,3 +82,31 @@ def goes_overlay(
         interpolation="nearest",
     )
     return
+
+
+def dropsondes_overlay(
+    dropsonde_ds,
+    ax,
+    variable="iwv",
+    variable_label=r"Integrated Water Vapor / kg m$^{-2}$",
+    colormap="Blues_r",
+):
+    from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+    im_launches = ax.scatter(
+        dropsonde_ds["lon"].isel(alt=10),
+        dropsonde_ds["lat"].isel(alt=10),
+        marker="o",
+        edgecolor="grey",
+        s=60,
+        transform=ccrs.PlateCarree(),
+        c=dropsonde_ds[variable],
+        cmap=colormap,
```

Comment:
Could you move these options to your function signature? This would allow users to modify, e.g. the color, to their liking

Reply:
done!